.github/workflows/pull_request.yml

```diff
@@ -63,7 +63,6 @@ jobs:
       - name: Run Python lint
         run: |
           pycodestyle --max-line-length=120 ./**/*.py
-          black --check --diff ./
```

Comment:
I would like to still have both, they target for different things and we just have to configure them properly. `black` ensures a common formatting (e.g. like `go fmt` a nd `pycodestyle` also checks some other code issues.